package.json

```diff
@@ -27,6 +27,7 @@
     "eslint-plugin-react": "^2.0.1",
     "eventemitter3": "^1.0.1",
     "express": "^4.11.2",
+    "extend": "^2.0.1",
```

Comment:
I would rather do not add any extend/underscore/object.assign libraries, If we can use immutable Seq or Map.